### PR TITLE
Patch cross thing trigger

### PR DESCRIPTION
**Pull request by yongpingchen**
- Make commandTarget field of CommandTriggerRequest required. Developer need to assign value for it. 
- Validate commandTarget field of  CommandTriggerRequest in TriggerOps
- Add description about cross thing trigger in PatchCommandTrigger method of APIAuthor and ThingIFAPI
- Modify description about cross thing trigger in PostCommandTrigger method of APIAuthor and ThingIFAPI

### CODE REVIEW

**Comment by yongpingchen** [comment]
@nfukuzaki can you review it?

**Comment by nfukuzaki** [comment]
LGTM

File: src/APIAuthor.ts [hunk 1]
```diff
@@ -230,9 +230,13 @@ export class APIAuthor {
     }
 
     /** Update a command trigger.
+     * When condition described by predicate is met, a registered command sends to thing related to target.
+     * `target` property and commandTarget in requestObject must belong to same owner.
+     *
      * @param {TypedID} target TypedID of target, only Types.THING is supported now.
      * @param {string} triggerID ID of trigger.
      * @param {Object} requestObject The fields of trigger to be updated.
+     *  If commandTarget property is not provided, `target` parameter is used as target of the command.
```
Comment (yongpingchen) [inline src/APIAuthor.ts]: will delete this description. commandTarget is required.

**Comment by yongpingchen** [comment]
@nfukuzaki can you review the implementation? So changes after spec review. Please check the description of PR.

**Pull request merged**